<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>270</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/backg.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>19</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/home.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Outlook</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(0, 0, 113, 0); 
border: none;
border-radius:10px;
color:white;}


QPushButton:hover{
background-color: rgb(255, 0, 0); 

}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>12</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/emm.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/em.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>1161</width>
      <height>871</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>-20</y>
        <width>1400</width>
        <height>871</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/baaak.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>160</y>
        <width>300</width>
        <height>187</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/sponn.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>-200</y>
        <width>481</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/aaa.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>160</y>
        <width>300</width>
        <height>187</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/specc.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>300</width>
        <height>187</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/stu.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>300</width>
        <height>187</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/emi.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>1191</width>
        <height>901</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/backkk.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>460</width>
        <height>720</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/Form1.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>-130</y>
        <width>481</width>
        <height>321</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>490</width>
        <height>325</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/aaa.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>310</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>371</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>371</width>
        <height>31</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>370</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>371</width>
        <height>31</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>590</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Id-sponsor</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>340</y>
        <width>35</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>131</width>
        <height>21</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>560</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Marque</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>490</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>duree fin</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>410</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>duree debut</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>660</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>630</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Prix-sponsor</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>730</y>
        <width>415</width>
        <height>45</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/Btn1.png&quot;);
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>440</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>520</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="ecran">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>1400</width>
        <height>841</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/baaak.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="ajoutersponsor">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>40</y>
        <width>200</width>
        <height>44</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/ajouter spon.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>130</y>
        <width>35</width>
        <height>36</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/recii.png&quot;);
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>130</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/klam.png&quot;);
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="tablespon">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>951</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView
{
	background-color: rgb(255, 255, 255);
	border: none;
	color: rgb(0, 0, 0);
}
QTableView::item:selected
{
	color:white;
	background-color: rgb(93, 93, 93);
	border: none;
}

QHeaderView::section
{
	border: none;
	background-color: qlineargradient(spread:pad, x1:0.989474, y1:0.977, x2:0.0214737, y2:0.085, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
	color: rgb(172, 172, 172);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ref">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/ref.png&quot;);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>501</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/chercher1.png&quot;);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="trier">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pddf">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(0, 0, 113, 0); 
border: none;
border-radius:10px;
color:white;}


QPushButton:hover{
background-color: rgb(255, 0, 0); 

}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>17</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/spon.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_spon">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(0, 0, 113, 0); 
border: none;
border-radius:10px;
color:white;}


QPushButton:hover{
background-color: rgb(255, 0, 0); 

}</string>
    </property>
    <property name="text">
     <string>Sponsors</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>13</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/stidio.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stidio">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(0, 0, 113, 0); 
border: none;
border-radius:10px;
color:white;}


QPushButton:hover{
background-color: rgb(255, 0, 0); 

}</string>
    </property>
    <property name="text">
     <string>Stidios</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>15</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/spectateur.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_spectateur">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(0, 0, 113, 0); 
border: none;
border-radius:10px;
color:white;}


QPushButton:hover{
background-color: rgb(255, 0, 0); 

}</string>
    </property>
    <property name="text">
     <string>Spectateurs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_empole">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(0, 0, 113, 0); 
border: none;
border-radius:10px;
color:white;}


QPushButton:hover{
background-color: rgb(255, 0, 0); 

}</string>
    </property>
    <property name="text">
     <string>Employe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>147</width>
      <height>147</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(&quot;C:/Users/ASUS/Desktop/QT_projet/dar.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
